src/audio/src/src.c

```diff
@@ -685,6 +685,10 @@ static int src_get_copy_limits(struct comp_data *cd,
 	s1 = cd->src.stage1;
 	s2 = cd->src.stage2;
 
+	/* Workaround for a pipeline issue in beginning of processing */
+	if (!source_get_frame_bytes(source))
```

Comment:
@singalsu lets take the real fix, the WA is just hiding the issue.

Reply:
Agree, it looks like SSP copier is clearing the source parameters in copy start.